app/src/main/java/ar/scacchipa/twittercloneapp/viewmodel/LoginViewModel.kt

```diff
@@ -0,0 +1,35 @@
+package ar.scacchipa.twittercloneapp.viewmodel
+
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class LoginViewModel: ViewModel() {
+
+    private val _mustShowErrorMsg = MutableLiveData(false)
```

Comment:
remember the pattern, private mutableLivedata, public liveData.

Reply:
It was replaced by:
    private val _mustShowErrorMsg = MutableLiveData(false)
    val mustShowErrorMsg = _mustShowErrorMsg as LiveData<Boolean>

    private val _navToFragAuthWeb = MutableLiveData(false)
    val navTpFragAuthWeb = _navToFragAuthWeb as LiveData<Boolean>